examples/bri-3/src/bri/identity/bpiAccounts/accounts.profile.ts

```diff
@@ -13,6 +13,7 @@ export class AccountsProfile extends AutomapperProfile {
   override get profile() {
     return (mapper) => {
       createMap(mapper, BpiAccount, BpiAccountDto);
+      createMap(mapper, BpiAccount, BpiAccount);
```

Comment:
I think we don't need this

Reply:
we need it, without it error that mapping between BpiAccount and BpiAccount doesn't exist is thrown